test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoCreateForSuiteRunner.java

```diff
@@ -49,6 +49,9 @@
 	private String nodes;
 	private String defaultNode;
 
+	//Use more initialBalance for this account as it is used as payer for the performance tests
+	private static OptionalLong initialBalance = OptionalLong.of(ONE_HBAR * 50000);
+
```

Comment:
This was not what we discussed. We want
```
CryptoCreateForSuiteRunner.initialBalance = 5 * LoadTest.initialBalance; // or 10 *
```
If these two balances don't link together, changing one might break the other in performance tests. It's harder to maintain them that way.